react.md

```diff
@@ -36,25 +34,25 @@ Try your best to answer each question on your own before looking up the answer o
       constructor(props){
         super(props)
         this.state = {
-          recipes: 
-            {name: 'Meatballs'},
-            {name: 'Mac & Cheese'}
-      
+          recipes:
+            name: 'Meatballs',
+            name: 'Mac & Cheese'
+
         }
       }
 
       render() {
-    
+
         return (
-    
-          let recipes = this.state.recipes.map(function(recipe){
+          let { recipes } = this.state
+          let newRecipes = recipes.map(function(recipe){
```

Comment:
I like that you destructured here. It cleans things up a bit. But give this question another look. Hint: Check syntax (ie. how the class is defined, how recipes is defined in the constructor)